src/components/Drawer/Drawer.tsx

```diff
@@ -32,13 +42,41 @@ const navItems = [
   },
 ];
 
-const drawerItems = (
+const externalNavItems = [
```

Comment:
a lot of these social or other official links will probably get pulled out into their own file so that they can be easily imported in several places and used. definitely a refactor-as-you-get-to-it kinda thing. not something required right now

Reply:
agreed